### PR TITLE
London-Class-7-Ali-Nosratipour-JavaScript-Core-3-Week-1

### PR DESCRIPTION
**Pull request by alinosratipour**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

**Review by yunusfirat — COMMENTED** [review]
Hi Ali, you have done great job. my suggestion would be to check if correct all requirement properly. when you add a new book and click checkbox it display on the screen as no which would have been expected to yes and when you change variable as library rather than mylibrary took too much time to change entire file. if I were you I would not change
let myLibrary = [ ]. please try to have a look one more time and if you can not fix then you can have look our class mates pullrequest to correct mistakes. All the best keep up good work.

**Comment by alinosratipour** [comment]
thank you for the feedback yunus you were right about changing library to myLibrary I made a mistake. but about a checkbox i double checked it it is doing what is suppose to do when i checked it as read it display yes and when is not checked it displays No.

File: mandatory/debugging-practice/script.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 
 window.addEventListener("load", function (e) {
   populateStorage();
   render();
 });
 
 function populateStorage() {
-  if (library.length == 0) {
+  if (myLibrary.length == 0) {
     let book1 = new Book("Robison Crusoe", "Daniel Defoe", "252", true);
```
Comment (NeginDn) [inline mandatory/debugging-practice/script.js]: There is a typo here. What was the name of the famous english novel?

**Review by NeginDn — COMMENTED** [review]
Hi Ali, 
Well done on picking all of the bugs, great eyes!  I only find one little thing to change and I have left a comment for you.

File: mandatory/debugging-practice/script.js (hunk 1 above)
Comment (alinosratipour) [reply mandatory/debugging-practice/script.js]: thank you negin for your feedback

**Pull request closed without merging**